website/content/en/docs/about/under-the-hood/guarantees.md

```diff
@@ -15,12 +15,22 @@ see which components support specific guarantees.
 
 ### At-least-once
 
-The **at-least-once** delivery guarantee ensures that an [event] received by a Vector component is
-ultimately delivered at least once. While rare, it is possible for an event to be delivered more
-than once. See the [Does Vector support exactly-once delivery?](#faq-at-least-once) FAQ below).
+The **at-least-once** delivery guarantee ensures that an [event]
+received by a Vector component is ultimately delivered at least
+once. For a source, when the `acknowledgements` option is enabled, this
+indicates that it will wait for _all_ connected sinks to either mark the
+event as delivered or to persist the events to a disk-based buffer
```

Comment:
I would change this to `durable buffer`.  In the future, external buffer implementations that provide durability as a first-class citizen will also acknowledge events in the same way as disk buffers.